common/djangoapps/coursetalk/migrations/0001_initial.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
```

Comment:
It might be best to delete this migration and regenerate it now that we're on Django 1.8 (sorry for the lengthy review process!).